src/modules/Content/index.tsx

```diff
@@ -58,16 +84,46 @@ export function ContentUI(props: ContentUIProps): ReactElement {
                                 <Input
                                     icon={<MagnifyingGlass />}
                                     name="search"
-                                    placeholder="Search Content..."
+                                    placeholder="Search Article..."
                                     value={searchTerm}
-                                    onChange={onChangeSearchTerm}
+                                    onChange={onChangeSearch}
                                 />
                             </>
                         ))
                         .with(ContentType.corporateActivity, () => onBack && backButton(ContentType.corporateActivity))
                         .exhaustive()}
                 </div>
             </div>
+            {searchTerm && (
```

Comment:
I don't have an issue w/ leaving this as is, but is the duplicating the code in the transcript module? should it be its own component?

Reply:
I agree with @lamflam that the UI piece is small enough that we don't necessarily need a separate component yet, but if we end up doing this a third time, then we should do it.